src/dune_rules/dune_file.mli

```diff
@@ -117,6 +117,43 @@ module Mode_conf : sig
 
     val eval : t -> has_native:bool -> Mode.Dict.Set.t
   end
+
+  module Lib : sig
+    type mode_conf = t
```

Comment:
You can do `type mode_conf := t` to avoid this alias from being exported.

Reply:
done in https://github.com/jchavarri/dune/pull/4/commits/99ef97ee7c78c79a0f47ba57362ead24b18d4356. I applied to another existing definition of `mode_conf` in `Mode_conf.Set`, hope it's fine.